docs/02-app/01-building-your-application/01-routing/03-linking-and-navigating.mdx

```diff
@@ -95,15 +93,24 @@ export function Navigation({ navLinks }) {
 }
 ```
 
-### Scrolling to an `id`
+#### Scrolling to an `id`
```

Comment:
You can also provide just the hash which will apply it to the same page `<Link href="#settings">`